linuxNew/jdk/suse/src/packageTest/java/packaging/SuseFlavours.java

```diff
@@ -32,13 +32,13 @@
 	public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
 		/*
 		 * OpenSUSE: All supported versions, see https://en.opensuse.org/Lifetime.
-		 *
-		 * SUSE itself is not distributed via Docker Hub, so we cannot test against it.
-		 */
+		 * SLES: All supported versions, see https://www.suse.com/lifecycle.
+		*/
 		return Stream.of(
 			Arguments.of("opensuse/leap", "15.1"),
 			Arguments.of("opensuse/leap", "15.2"),
-			Arguments.of("opensuse/leap", "15.3")
+			Arguments.of("opensuse/leap", "15.3"),
+			Arguments.of("registry.suse.com/suse/sle15", "latest")
```

Comment:
Should also add sle12? `Arguments.of("registry.suse.com/suse/sle12", "latest")`

Reply:
I tried adding sles12 but I needed a subscription to even run zypper update.

Reply:
Got it. We probably need to add some comment or open an issue to make a note of that. As this means we release installer on sles12 without any tests.